style.css

```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;400;900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;900&display=swap");
+
+.homepage-content-container {
+  margin: 3rem;
+  padding: 2.5rem 4rem 4rem 9rem;
+}
+
+.homepage-title {
+  font-family: "Playfair Display", serif;
+  font-size: 6rem;
+  font-weight: 800;
```

Comment:
You are not importing a font with a weight of 800. So, I think it will not apply.

Reply:
done